faq.md

```diff
@@ -111,7 +111,7 @@ built with Flutter to users. We'd love to hear what you're building!
 ### What is inside the Flutter SDK?
 
 * Heavily optimized, mobile-first 2D rendering engine with excellent support for text
-* Functional-reactive framework
+* Reactive framework
```

Comment:
We're no more reactive than functional, so I disagree that this is a correct change.